configs/components/lpj_guess/lpj_guess.yaml

```
@@ -0,0 +1,203 @@
+    comp_command: "mkdir -p build; cd build; cmake ..; make -j8"
+
+    install_bins: "build/guess"
+    clean_command: "rm -rf build"
+    executable: guess
+    execution_command: ${executable} ${runscript_name}
+    #resolution: TL159 #if it's not set i dosen't work! 
+    resolution: TL255 #if it's not set i dosen't work!
+
+    choose_version:
+        # Here you can place all the variables you would want to control via the version
+        # variable above (for example, you could also add ``git-repository``,
+        # ``comp_command``, ``destination``, ...) 
+        4.1.1:
+            runscript_name: run_trunk_4_1.ins
+            git-repository: https://gitlab.dkrz.de/ec-earth/lpj-guess4.1.1.git
+            branch: main
+        4.1.2:
+            runscript_name: run_coupled_4_1_2.ins
+            git-repository: https://git.smhi.se/e8891/lpjg-4.1.git
+            branch: lpj_guess_awiesm3
+            mygrid: TL255
+            install_bins: build/guess_TL255
+            executable: guess_TL255
+
+                     
+    # ``model_dir`` let's ESM-Tools know where is the source code of your component (must
+    # be an absolute path). Normally, changed via the runscript. We recommend you leave it
+    # as below, changing only the ``name_of_your_model`` part to whatever you defined as 
+    # the ``model`` var. This will automatically select the source code for esm_master if
+    # the command is operating with a coupled setup
+    model_dir: ${general.model_dir}
+
+    # This section takes care of dealing with environment changes you might want to do, to
+    # deviated from the defaults define for the specific machine
+    # (``configs/machine/<the_machine_used>.yaml``). For more information consult:
+    # https://esm-tools.readthedocs.io/en/latest/esm_environment.html#esm-environment
+    choose_computer.name:
+        levante:
+            pool_dir: "/work/ab0246/a270092/input/"
+
+    # Some recommended defaults (restart for old experiment yes or no?)
+    lresume: false
+    time_step: 86400
+
+    choose_lresume:
+        false:
+            restart_num: 0
+        true:
+            restart_num: 1
+
+    # Some commonly used variables in other ESM-Tools components (but optional)
+    input_dir: ${pool_dir}/lpj-guess/
+    namelist_dir: ${general.esm_namelist_dir}/lpj_guess/
+    gridlist_dir: ${input_dir}/gridlists/
+    forcing_dir: ${input_dir}/climate_forcing_data/
+
+    #gridlist: "full_global" # sets up grid cell list =/ setup_name!!
+    gridlist: "home" # sets up grid cell list =/ setup_name!!
+    setup_name: global # sets up all the model parameters
+
+
+    # File dictionaries to control the copying/moving/linking of all files associated to
+    # the simulation. This syntax will change for something better soon. For more details
+    # about this syntax consult:
+    # https://esm-tools.readthedocs.io/en/latest/yaml.html#file-dictionaries
```

```suggestion
    # File dictionaries
```

Do you want to keep the whole comment, or would a small comment just do?

---

Small comment is enough